src/webgpu/api/validation/encoding/cmds/render/draw.spec.ts

```diff
@@ -7,120 +7,45 @@ and parameters as expect.
 import { makeTestGroup } from '../../../../../../common/framework/test_group.js';
 import { ValidationTest } from '../../../validation_test.js';
 
-// Class that indicate how to call a draw function
-class DrawCall {
-  test: ValidationTest;
-  drawType: 'draw' | 'drawIndexed' | 'drawIndirect' | 'drawIndexedIndirect' = 'draw';
-
-  // Draw
-  vertexCount: number = 4;
-  firstVertex: number = 0;
-
-  // DrawIndexed
-  indexCount: number = 4;
-  firstIndex: number = 0;
-  baseVertex: number = 0;
-
-  // Both Draw and DrawIndexed
-  instanceCount: number = 1;
-  firstInstance: number = 0;
-
-  constructor(test: ValidationTest) {
-    this.test = test;
-  }
+interface DrawIndexedParameter {
+  indexCount: number;
+  firstIndex?: number;
+  baseVertex?: number;
+  instanceCount?: number;
+  firstInstance?: number;
+}
 
-  callDraw(encoder: GPURenderEncoderBase) {
-    switch (this.drawType) {
-      case 'draw': {
-        encoder.draw(this.vertexCount, this.instanceCount, this.firstVertex, this.firstInstance);
-        break;
-      }
+class F extends ValidationTest {
+  callDrawIndexed(
```

Comment:
nit: I (personally) think this should be a standalone function (`function callDrawIndexed`) rather than a member of the fixture. It makes it a bit lighter weight and if you do this you won't have to customize the fixture.

Reply:
Done.